test/acceptance/specs/MockDataSugar.scala

```diff
@@ -441,6 +442,89 @@ trait MockDataSugar {
 
   val applicationToBlock = applicationToDelete
 
+  val applicationToUnblock =
```

Comment:
Could use blockedApplication

Reply:
I have changed the way these apps are created. Have a look to see if it's still a problem.